udata/core/dataset/events.py

```diff
@@ -23,7 +23,7 @@ def serialize_resource_for_event(resource):
         'checksum_type': resource.checksum.type if resource.checksum else None,
         'checksum_value': resource.checksum.value if resource.checksum else None,
         'created_at': to_iso_datetime(resource.created_at),
-        'modified': to_iso_datetime(resource.modified)
+        'modified': to_iso_datetime(resource.last_modified)
```

Comment:
Do we want to modify it to `last_modified` (on the reception side too)?

Reply:
well `resource.modififed` won't exist anymore so we have to either modify it to `last_modified` or `last_modified_internal`

Reply:
I meant the left-hand part, `'modified'` in `resource_dict`

Reply:
Well I guess we will have to modify it on the search service then ;)

Reply:
Could you prepare a PR there please? :pray: